python/python/lance/dataset.py

```diff
@@ -418,6 +425,8 @@ def to_table(
             Return row ID.
         use_stats: bool, default True
             Use stats pushdown during filters.
+        full_text_query: (str, str), optional
```

Comment:
This documentation is a little confusing.  It seems to imply the input can be a tuple `(str, str)` but in the code it looks like you accept `str` or `dict`.  However, later in the rust code, it looks like you only really accept `dict`.

If that's the case we should change this to `dict` and be clear what the keys are and what each one means.  I think, from inspection, you want a `query` key (the query string) and a `columns` key (the columns to search).

However, it's also not clear what kind of search is performed.  Is this a "contains" query?  Is it searching for all strings that contain the given query string?  Can the query string have spaces?  If it does are these treated as multiple words or a combined string?  For example, will the query "a b" match the string "b a"?

Reply:
Fixed that doc, it would accept `str` or `dict`

it would search for documents that contains any words of `query` string, and rank them with BM25. Say "a b" can match docs contain "a" or "b" or "b a"